src/functionalTest/resources/features/F-008/S-071.v1_1.td.json

```diff
@@ -0,0 +1,153 @@
+{
+  "_guid_": "S-071.v1_1",
+  "_extends_": "S-071",
+
+  "expectedResponse": {
+    "body": {
+      "roleAssignmentResponse": [
+        {
```

Comment:
Can we move all roleAssignments except "tribunal-caseworker" into `S-071.td.json`